.circleci/config.yml

```
@@ -3,6 +3,16 @@ version: 2.1
 orbs:
   architect: giantswarm/architect@5.6.0
 
+jobs:
+  template-chart:
+    docker:
+      - image: giantswarm/helm-chart-testing:v3.11.0
+    steps:
+      - checkout
+      - run: cd helm/logging-operator && helm dep up
+      - run: helm template helm/logging-operator
```

This could be a single run I believe
```suggestion
      - run: |-
          cd helm/logging-operator
          helm dep up
          helm template helm/logging-operator
```